scripts/enforce-property-single-quote-in-bundle-message

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
Whoops... non-executable file (do you have `pre-commit` hook set locally?)

Reply:
Yes, I have but this is specific for my local setup - I have our plugin repository on `Bitlocked` partition and access to this partition on Linux is a bit tricky (need to use dislocker and some additional virtual device/partition). Default mount options cause that I can't set `x` permission to any file on this partition (and then I must remember to execute `git update-index --chmod=+x <file-path>` to update permissions in a git repo).
Some day I must just tweak mount point options to fix this  :)

Reply:
Wowow but can you then run e.g. `./gradlew` or get any git hook executed? :hushed: since there's no way any file is executable on this partition